Product Management Service/Code/product_management_service.sql

```diff
@@ -2,16 +2,10 @@ CREATE TABLE products (
   id int PRIMARY KEY,
   name varchar(255),
   price decimal(10,2),
-  category varchar(255) FOREIGN KEY REFERENCES category(product_category)
-);
-
-CREATE TABLE product_details (
-  id int PRIMARY KEY,
+  category varchar(255),
   image blob,
-  description text
+  description text,
+  specification JSON,
+  created_on timestamp,
+  last_updated timestamp 
```

Comment:
naming should be what this is, basically created_timestamp and updated_timestamp